src/pages/iv-items/forms/PhysicalForm.js

```diff
@@ -219,8 +228,8 @@ const PhysicalForm = ({ labels, editMode, maxAccess, store }) => {
                 value={formik.values.diameter}
                 maxAccess={maxAccess}
                 readOnly={formik.values?.shape === 1}
-                onChange={e => handleFieldChange('diameter', 4, e.target.value)}
-                onClear={() => handleFieldChange('diameter', 4, 0)}
+                onMouseLeave={e => handleFieldChange('diameter', 4, e?.target?.value)}
```

Comment:
you can simplify your code by passing e (event) directly and then assigning e?.target?.value to value inside the handleFieldChange function. ex:  const handleFieldChange = (fieldName, e) => {
  const value = e?.target?.value; }